### PR TITLE
Trophy Fishing Tab

### PR DESCRIPTION
**Pull request by efefury**
- auto disable NEU map on start up
- small stuff
- add trophy fishing tab in pv

### CODE REVIEW

**Review by DeDiamondPro — DISMISSED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/commands/dungeon/MapCommand.java [hunk 1]
```diff
@@ -50,6 +50,12 @@ public MapCommand() {
 
 	@Override
 	public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+		if (!NotEnoughUpdates.INSTANCE.config.hidden.dev) {
+			Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(
+				EnumChatFormatting.RED + "[NEU] The map does not work right now. You can use the map from other mods, for example: SkyblockAddons, DungeonsGuide, Skytils "));
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/dungeon/MapCommand.java:55]:
```suggestion
				EnumChatFormatting.RED + "[NEU] The map does not work right now. You can use the map from other mods, for example: SkyblockAddons, DungeonsGuide, or Skytils "));
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/dungeon/MapCommand.java [hunk 2]
```diff
@@ -50,6 +50,12 @@ public MapCommand() {
 
 	@Override
 	public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+		if (!NotEnoughUpdates.INSTANCE.config.hidden.dev) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/dungeon/MapCommand.java:53]: if there is an exception for dev mode here shouldn't there also be an exception for it when checking to override the config?

**Review by IRONM00N — APPROVED** [review]
thanks <3

**Review by DeDiamondPro — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]

**Comment by IRONM00N** [comment]
admin abuse :)

**Pull request merged**